experimental/PropertyDDS/examples/property-inspector/src/containerCode.ts

```diff
@@ -4,10 +4,10 @@
  */
 
 import { ModelContainerRuntimeFactory, getDataStoreEntryPoint } from "@fluid-example/example-utils";
-import { IContainer } from "@fluidframework/container-definitions";
-import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import type { IContainer } from "@fluidframework/container-definitions";
+import type { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
```

Comment:
While I was merging in VS Code, the eslint plugin was busy autofixing things.

I guess we'll keep the changes, even though it makes this PR noisier than it needs to be.